src/main/java/graphql/schema/GraphqlTypeComparatorEnvironment.java

```diff
@@ -20,7 +19,6 @@
     private Class<? extends GraphQLSchemaElement> elementType;
 
     private GraphqlTypeComparatorEnvironment(Class<? extends GraphQLSchemaElement> parentType, Class<? extends GraphQLSchemaElement> elementType) {
-        Assert.assertNotNull(elementType, () -> "elementType can't be null");
```

Comment:
This needs to be relaxed because now we can have a `GraphqlTypeComparatorEnvironment` where there is only a parent type and not element type.  

This was probably wrong in the oder code but because we never sorted the top level types we never noticed this.